tags/TagMap.md

```diff
@@ -23,23 +23,35 @@ and group stats, annotate traces and logs.
 `TagValue` is a string. It MUST contain only printable ASCII (codes between
 32 and 126)
 
-## TagScope
+## TagMetadata
 
-`TagScope` is used to determine the scope of a `Tag`. The values for the `TagScope` are
-Local or Request. In future, additional values can be added to address specific situations.
+`TagMetadata` contains properties associated with a `Tag`. For now the only property `TagTTL`
+is defined. In future, additional properties may be added to address specific situations.
 
-The tag creator determines the scope of the tag.
+A tag creator determines metadata of a tag it creates.
 
-**Local Scope**
-Tag with `Local` scope are used within the process it created. Such tags are not propagated
-across process boundaries. Even if the process is reentrant the tag MUST be excluded from
-propagation when the call leaves the process.
+### TagTTL
 
-**Request Scope**
-If a tag is created with the `Request` scope then it is propagated across process boundaries subject 
-to outgoing and incoming (on remote side) filter criteria. See `TagPropagationFilter` in 
-[Tag Propagation](#Tag Propagation). Typically 'Request' tags represents a request, processing
- of which may span multiple entities.
+`TagTTL` is an integer that represents number of hops a tag can propagate. Anytime a tag is propagated
+over a carrier like http, grpc, etc it is considered to have traveled one hop. 
+
+- A receiver MUST decrement the value of `TagTTL` by one if it is greater than zero.
+- A receiver MUST discard the `Tag` if the `TagTTL` value is zero.
+- A receiver MUST not change the value of `TagTLL` if it is -1.
```

Comment:
I don't think this is true. Receiver can overwrite any `TagTTL` value locally and propagation should respect the local value.

Reply:
The above was meant for when tag is received. Later in conflict resolution it does say local tags can override received tags or other local tags created earlier.  I agree that it is misleading.

How about I rephrase the above to 

"
Upon receiving tag from remote entity a receiver
- MUST decrement the value of `TagTTL` by one if it is greater than zero.
- MUST discard the `Tag` if the `TagTTL` value is zero.
- MUST not change the value of `TagTLL` if it is -1.
"
Does that sound ok?

Reply:
How about replacing "receiver" with "propagator"?

Reply:
"propagator" could be a proxy in the middle, no? Proxy probably should not affect TTL.

Reply:
I'm referring to the component that handles the encoding/decoding of TTL info. Agree that "propagator" may not be the best term here. How about "tag serializer"?

Reply:
@songy23 'propagator' sounds good.

@yurishkuro are you referring to a http-proxy or a proxy that may simply extract tags/tracecontext and inject into same or another format? If it is later I can add something like "If propagator is acting as a proxy then it MUST leave TagTTL unchanged".